Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverComponent.cs

```diff
@@ -151,6 +154,14 @@ public void ClearProvider()
             HasApcPower = false;
         }
 
+        public bool TryGetWireNet([NotNullWhen(true)] out INodeGroup nodeGroup)
+        {
+            var wireNet = Provider.GetWireNet();
+
+            nodeGroup = wireNet;
+            return wireNet != default;
+        }
+
```

Comment:
This method doesn't use any non-public information, and is only used by one other class,  so I don't think this method should be on this class. Also, you may want to check NeedsProvider, otherwise you may return an instance of PowerProviderComponent.NullProvider which may cause problems with whatever uses the returned value.